internal/http/handler.go

```diff
@@ -33,6 +36,20 @@ func NewHandler(us models.UserService, as models.ApplicationService, mailer mail
 
 	r.Mount("/", h.WebHandler)
 
+	r.Get("/robots.txt", h.getRobotsTxt())
+
 	h.Mux = r
 	return &h
 }
+
+// getRobotsTxt serves the plain text robots.txt file for web crawlers.
+// The file itself is defined as a string in the handler.
+func (h *Handler) getRobotsTxt() http.HandlerFunc {
+	robotsTxt := `User-agent: *
+Disallow:
```

Comment:
Fix this spacing

Reply:
multiline strings in Go are kinda wack, If we line add white space for the second and third lines then that shows up in the string and the robots.txt parsers might have trouble with it.  I tried making it a little less awful though